code/game/objects/structures/tables_racks.dm

```diff
@@ -406,19 +406,25 @@
 			qdel(W)
 			return
 
+	//these should maybe be moved into something like W.table_act()
 	if (W.is_wrench(user) && can_disassemble())
 		to_chat(user, "<span class='notice'>Now disassembling table...</span>")
 		W.playtoolsound(src, 50)
 		if(do_after(user, src,50))
 			destroy()
 		return
 
-	if(istype(W, /obj/item/weapon/cookiesynth))
+	if (istype(W, /obj/item/weapon/reagent_containers/pan) && user.a_intent == I_DISARM)
```

Comment:
Be sure to document this on the intents page. Also unsure of why this should be tied to an intent. It may make more sense as a mousedrop action like bags.